aiida/schedulers/plugins/slurm.py

```diff
@@ -707,53 +707,68 @@ def _parse_kill_output(self, retval, stdout, stderr):
 
         return True
 
-    def parse_output(self, detailed_job_info, stdout, stderr):  # pylint: disable=inconsistent-return-statements
+    def parse_output(self, detailed_job_info=None, stdout=None, stderr=None):
```

Comment:
Just mentioning that we don't necessarily need to provide default values here (but I guess this makes it explicit that these parameters can be `None`)